packages/fiori3/src/components/ResponsivePopover/index.tsx

```diff
@@ -1,12 +1,13 @@
 import { Device, Event } from '@fiori-for-react/utils';
-import React, { Component } from 'react';
 import { Dialog } from '../../lib/Dialog';
 import { Popover } from '../../lib/Popover';
+import React, { Component, CSSProperties } from 'react';
 import { DialogPropTypes } from '../../webComponents/Dialog';
```

Comment:
import from `lib`

Reply:
how is it possible to extend PropTypes from a lib export?

Reply:
Yes, you are right - sorry for that